src/main/java/edu/ucla/library/bucketeer/verticles/ImageWorkerVerticle.java

```diff
@@ -60,4 +111,44 @@ protected Logger getLogger() {
         return LOGGER;
     }
 
+    /**
+     * This message sender is specifically for item processing requests; it behaves differently from the default message
+     * sender that our other handlers use.
+     *
+     * @param aJsonObject A message with information about the image to upload
+     * @param aListener The destination of the message
+     * @param aPromise A promise that the JP2/JPX image has been uploaded
+     */
+    private void sendImageRequest(final String aListener, final JsonObject aJsonObject, final Promise<Void> aPromise) {
```

Comment:
I'll look again. I thought about it, but thought there was some difference so just punted on the idea but will give it a second look, thanks.

Reply:
Or it may have been the handler/verticle difference now that I think about it. Other than shunting it off into a utilities class I didn't see an immediate solution since they extend different Abstract classes.

Reply:
I created a ticket for this with some ideas on how to reduce this sort of code duplication: https://jira.library.ucla.edu/browse/IIIF-992